spec/spec.go

```diff
@@ -17,25 +17,32 @@ package spec
 import (
 	"encoding/json"
 	"sort"
+
+	"github.com/jsonnet-bundler/jsonnet-bundler/spec/deps"
 )
 
 // JsonnetFile is the structure of a `.json` file describing a set of jsonnet
 // dependencies. It is used for both, the jsonnetFile and the lockFile.
 type JsonnetFile struct {
-	Dependencies map[string]Dependency
+	// whether to use a go-like package names (github.com/user/repo)
+	GoImportStyle bool
```

Comment:
IMHO we can rename to something better. Maybe `absoluteImports` or even something like `imports: absolute` and `imports: free`.

Reply:
Yeah that's cool. I think we an go with `absoluteImports: true`. Using `imports: <style>` feels overkill, as we most probably won't add any other import style soon, or it could become very messy for users.